lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -6,7 +6,9 @@
                 errorMessage: gettext('An error has occurred. Please try again.'),
 
                 srAddBookmarkText: gettext('Click to add'),
+                bookmarkText: gettext('Bookmark This Page'),
```

Comment:
Why is this the only string that is in title case? @catong should this be in sentence case like the other strings?